.github/workflows/pasmanteria-fryderyka.yml

```diff
@@ -0,0 +1,63 @@
+name: Pasmanteria Fryderyka CI/CD
+
+on:
+  push:
+    branches: ["main"]
+    paths: ["src/**"]
+  pull_request:
+    branches: ["main"]
+    paths: ["src/**"]
+
+env:
+    REGISTRY: ghcr.io
+    IMAGE_NAME: KanarekLife/pasmanteria_fryderyka
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+            packages: write
+            attestations: write
+            id-token: write
+        steps:
+            - uses: actions/checkout@v4
+            - name: Log in to the Container Registry
+              if: github.event_name == 'push'
+              uses: docker/login-action@v3.3.0
+              with:
+                registry: ${{ env.REGISTRY }}
+                username: ${{ github.actor }}
+                password: ${{ secrets.GITHUB_TOKEN }}
+            - name: Extract metadata (tags, labels) for Docker
+              id: meta
+              uses: docker/metadata-action@v5.6.1
```

Comment:
Will this step add the `latest` tag on the `main` branch? I think it will be needed in the `docker-compose.yaml`.

Reply:
it should